Build Failed

Log:
```plain
cd src/ && ( test -e Makefile.CCR-Plus || /usr/bin/qmake -o Makefile.CCR-Plus /tmp/CCR-Plus/src/CCR-Plus.pro ) && make -f Makefile.CCR-Plus 
make[1]: Entering directory '/tmp/CCR-Plus/src'
g++ -c -pipe -O2 -std=gnu++11 -Wall -Wextra -D_REENTRANT -fPIC -DQT_NO_DEBUG -DQT_WIDGETS_LIB -DQT_GUI_LIB -DQT_XML_LIB -DQT_CORE_LIB -I. -isystem /usr/include/qt -isystem /usr/include/qt/QtWidgets -isystem /usr/include/qt/QtGui -isystem /usr/include/qt/QtXml -isystem /usr/include/qt/QtCore -I. -I. -I/usr/lib/qt/mkspecs/linux-g++ -o configuredialog.o configure/configuredialog.cpp
In file included from ./common/global.h:6,
                 from configure/configuredialog.cpp:4:
./common/contest.h: In member function ‘void Contest::SortPlayers(bool (*)(const Player*, const Player*))’:
./common/contest.h:47:112: warning: ‘void qSort(RandomAccessIterator, RandomAccessIterator, LessThan) [with RandomAccessIterator = QList<Player*>::iterator; LessThan = bool (*)(const Player*, const Player*)]’ is deprecated: Use std::sort [-Wdeprecated-declarations]
   47 |     void SortPlayers(bool (*cmp)(const Player* x, const Player* y)) { qSort(players.begin(), players.end(), cmp); }
      |                                                                                                                ^
In file included from /usr/include/qt/QtCore/qlist.h:43,
                 from /usr/include/qt/QtGui/qicon.h:45,
                 from /usr/include/qt/QtWidgets/qabstractbutton.h:44,
                 from /usr/include/qt/QtWidgets/qcheckbox.h:44,
                 from /usr/include/qt/QtWidgets/QCheckBox:1,
                 from configure/configuredialog.cpp:1:
/usr/include/qt/QtCore/qalgorithms.h:181:46: note: declared here
  181 | QT_DEPRECATED_X("Use std::sort") inline void qSort(RandomAccessIterator start, RandomAccessIterator end, LessThan lessThan)
      |                                              ^~~~~
configure/configuredialog.cpp: In member function ‘bool ConfigureDialog::apply()’:
configure/configuredialog.cpp:55:52: error: invalid use of incomplete type ‘class QHeaderView’
   55 |         int t = configure_table->horizontalHeader()->logicalIndex(i);
      |                                                    ^~
In file included from /usr/include/qt/QtWidgets/QTableView:1,
                 from ./configure/configuretable.h:4,
                 from configure/configuredialog.cpp:6:
/usr/include/qt/QtWidgets/qtableview.h:50:7: note: forward declaration of ‘class QHeaderView’
   50 | class QHeaderView;
      |       ^~~~~~~~~~~
configure/configuredialog.cpp: In member function ‘void ConfigureDialog::on_pushButton_adv_clicked()’:
configure/configuredialog.cpp:87:52: error: invalid use of incomplete type ‘class QHeaderView’
   87 |         int t = configure_table->horizontalHeader()->logicalIndex(i);
      |                                                    ^~
In file included from /usr/include/qt/QtWidgets/QTableView:1,
                 from ./configure/configuretable.h:4,
                 from configure/configuredialog.cpp:6:
/usr/include/qt/QtWidgets/qtableview.h:50:7: note: forward declaration of ‘class QHeaderView’
   50 | class QHeaderView;
      |       ^~~~~~~~~~~
make[1]: *** [Makefile.CCR-Plus:1166: configuredialog.o] Error 1
make[1]: Leaving directory '/tmp/CCR-Plus/src'
make: *** [Makefile:49: sub-src-CCR-Plus-pro-make_first] Error 2
```

Qt: 5.15.0
GCC: 10.1.0
Linux

--- a/src/configure/advancedconfiguredialog.h
+++ b/src/configure/advancedconfiguredialog.h
@@ -2,6 +2,7 @@
 #define ADVANCEDCONFIGUREDIALOG_H
 
 #include <QDialog>
+#include <QHeaderView>
 
 class QAbstractButton;
 class Problem;

